Makefile

```diff
@@ -13,3 +13,18 @@ dev: ## Run local server to check you content while writing
 
 build: ## Build the static files of the website
 	hugo
+
+docker-base: 
+	docker build -f Dockerfile.base -t baker-docs:base .
+
+docker-dev: docker-base ## Use docker for baker website development
+	docker run -w /baker -v $$PWD:/baker -p 1313:1313 baker-docs
```

Comment:
```suggestion
	docker run -w /baker -v $$PWD:/baker -p 1313:1313 baker-docs:base hugo server --bind=0.0.0.0
```

Reply:
it doesn't work without this change

Reply:
I just forgot to add the entry-point in Dockerfile.base. done